### ISSUE TITLE
Show only errors and warnings

### ISSUE THREAD

**Issue by rhwood** [OSSIndex/ossindex-maven-plugin#16]
Can a configuration option (like "quiet") be added to only show errors and warnings in the standard maven output?

If such an option is made available, it would nice to have a single line summary to show that plugin ran (something like ```Audited dependencies: 14, Errors: 0, Warnings: 0, Ignored: 0, Time elapsed: 5.3 s```).

**Comment by OSSIndex-Admin** [comment]
I am not really sure what would qualify as a warning. I have written some code (not published to central yet -- not tested fully) which writes vulnerabilities to the log, and reports as follows:

```
[INFO] Audited dependencies: 59, Vulnerabilities: 0, Ignored: 0, Time elapsed: 1991 ms
```

See [this commit](https://github.com/OSSIndex/ossindex-maven-plugin/commit/449bc7839ec8cdb4fa277ec509297fb5574bbf3c)

As mentioned, I still have testing a some other features to go in, but this should be published in reasonably short order.

**Comment by OSSIndex-Admin** [comment]
Still needs testing and not published.

**Comment by rhwood** [comment]
I can't think of any differences between "warning" and "ignore" at this time other than in a summary like this, a "warning" might be for any "we were supposed to check but didn't" situation, while "ignored" is you told us not to check.

You might also want to warn on ignored dependencies that are not actually dependencies (you instructed me to ignore foo:bar:3.0.0, but do not use it).

**Comment by OSSIndex-Admin** [comment]
The -Daudit.quiet option will cause only the errors (and soon, warnings) to be logged to stdout, along with an [Info] summary line.

Printing warnings on ignored issues will be released shortly.

This is fixed in release 2.3.5 (published on maven central).
